src/code/z_view.c

```diff
@@ -403,7 +424,10 @@ s32 func_800AB0A8(View* view) {
     return 1;
 }
 
-s32 func_800AB2C4(View* view) {
+/**
+ * Apply scissor, viewport and projection (ortho) to OVERLAY_DISP.
+ */
+s32 View_ApplyOrthoToOverlay(View* view) {
```

Comment:
I would go for an even longer name and name this `View_ApplyOrthoToOverlayDisp` to prevent confusion with a code overlay